docs/configuration.md

```diff
@@ -146,23 +159,25 @@ The `settings` for each analyzer in the `.lookout.yml` config file will be merge
 - Arrays are replaced
 - Null value replaces object
 
-### Advanced fine-tuning
+## Timeouts
+
+The timeouts used by **lookout** for some operations can be modified or disabled from the `config.yml` file.
 
-The configuration file also provides the possibility to change default timeouts.
+If any timeout is set to `0`, there will be no timeout for that process.
 
-Below is the list of different timeouts with their default values:
+Below is the list of different timeouts in **source{d} lookout**, with their default values:
 
 ```yaml
 # These are the default timeout values. A value of 0 means no timeout
 timeout:
-  # Timeout for an analyzer response to a NotifyReviewEvent
+  # Timeout for an analyzer to response a NotifyReviewEvent
```

Comment:
I'd leave "Timeout for an analyzer response to a NotifyReviewEvent". Or change this to "to reply".